.github/workflows/package_installers.yml

```diff
@@ -115,3 +116,33 @@ jobs:
             app/pkg/build/jpackage/*.dmg
             app/pkg/build/jpackage/*.pkg
             app/pkg/build/jpackage/*.deb
+
```

Comment:
Since `actions/upload-release-asset@v1` doesn't support wildcards for the `asset_path` argument and the installer's filename is different across operating systems, I had to write my own script.